runtime/compiler/codegen/CodeGenGPU.cpp

```diff
@@ -687,10 +688,10 @@ static void getNodeName(TR::Node* node, char * s, TR::Compilation *comp)
                int64_t                 doubleBits;
                };
             doubleValue = node->getFloat();
-            len = snprintf(s, MAX_NAME, "0x%0.16llx", doubleBits);
+            len = snprintf(s, MAX_NAME, "0x%016" OMR_PRIx64, doubleBits);
```

Comment:
For this and many other places touched in this PR, would it make sense to use `#` as the flag in the format specifier and eliminate the explicit `0x` in the string?   In this particular instance, `"%#016" OMR_PRIx64`)

Reply:
Sorry for the scope creep.  But since you're changing the format specifier anyway it seems like an opportune moment to do this.

Reply:
That would make the `0x` prefix conditional, a change that I think belongs in a separate pull-request.